### ISSUE TITLE
How to understand the output of the obb2xyxy function

### ISSUE THREAD

**Issue by canoe-Z** [open-mmlab/mmrotate#129]
When I call the ``obb2xyxy`` function like this,
```
obb = torch.tensor([100, 50, 200, 100, 0]).unsqueeze(0)
print(obb2xyxy(obb,'le90'))
```
The output is ``tensor([[  0.,   0., 200., 100.]])``. Obviously the point ``(0,0)`` is not th left-**top** point. In this case, I don't get the expected output ``tensor([[  0.,   100., 200., 0.]])`` as the docstring said formatted by``[x_lt,y_lt,x_rb,y_rb]``. What's wrong with that?

**Comment by yangxue0827** [comment]
left-top point: (100, 50) - (200, 100) / 2 = (0, 0)
right-bottom point: (100, 50) + (200, 100) / 2 = (200, 100)
I think the output is right.

**Comment by canoe-Z** [comment]
Thanks so much for your reply to this stupid quesiton. I forget that the y-axis direction is reversed in the images.

> left-top point: (100, 50) - (200, 100) / 2 = (0, 0) right-bottom point: (100, 50) + (200, 100) / 2 = (200, 100) I think the output is right.
